base/applications/regedit/resource.h

```diff
@@ -73,48 +72,19 @@
 
 #define ID_HELP_HELPTOPICS            32794
 #define ID_HELP_ABOUT                 32795
-#define ID_WINDOW_CASCADE             32797
-#define ID_WINDOW_TILE                32798
-#define ID_WINDOW_ARRANGEICONS        32799
-#define ID_OPTIONS_FONT               32800
-#define ID_OPTIONS_AUTOREFRESH        32801
-#define ID_OPTIONS_READONLYMODE       32802
-#define ID_OPTIONS_CONFIRMONDELETE    32803
-#define ID_OPTIONS_SAVESETTINGSONEXIT 32804
-#define ID_SECURITY_PERMISSIONS       32805
-#define ID_VIEW_TREEANDDATA           32806
-#define ID_VIEW_TREEONLY              32807
-#define ID_VIEW_DATAONLY              32808
-#define ID_VIEW_DISPLAYBINARYDATA     32810
-#define ID_VIEW_REFRESHALL            32811
-#define ID_VIEW_REFRESHACTIVE         32812
-#define ID_VIEW_FINDKEY               32813
-#define ID_TREE_EXPANDONELEVEL        32814
 #define ID_TREE_EXPANDBRANCH          32815
-#define ID_TREE_EXPANDALL             32816
 #define ID_TREE_COLLAPSEBRANCH        32817
-#define ID_EDIT_ADDKEY                32818
-#define ID_EDIT_ADDVALUE              32819
-#define ID_EDIT_BINARY                32821
-#define ID_EDIT_STRING                32822
-#define ID_EDIT_DWORD                 32823
-#define ID_EDIT_MULTISTRING           32824
 #define ID_REGISTRY_OPENLOCAL         32825
-#define ID_REGISTRY_CLOSE             32826
 #define ID_REGISTRY_LOADHIVE          32827
 #define ID_REGISTRY_UNLOADHIVE        32828
-#define ID_REGISTRY_RESTORE           32829
-#define ID_REGISTRY_SAVEKEY           32830
 #define ID_REGISTRY_SELECTCOMPUTER    32831
```

Comment:
Also unused; has been replaced by `ID_REGISTRY_(DIS)CONNECTNETWORKREGISTRY`

Reply:
I acted on that suggestion by https://github.com/reactos/reactos/pull/7194/commits/09e7bdc46edbc5d8c3d5000e84121cea9eedfca0 -> Conversation resolved, and re-review requested hereby.